docs/topics/features.md

```diff
@@ -170,6 +170,25 @@ Per default Calico still allows all communication within the cluster. Using Kube
 - [NetworkPolicy Example Walkthrough](https://kubernetes.io/docs/getting-started-guides/network-policy/walkthrough/)
 - [Calico Kubernetes](https://github.com/Azure/aks-engine/blob/master/examples/networkpolicy)
 
+### Calico 3.3 cleanup after upgrading to 3.5
+
+Because Calico 3.3 is using Calico CNI, while Calico 3.5 moves to Azure CNI, if the cluster is upgraded from calico 3.3 to 3.5, then some manual cluster resource cleanup will be required to successfully complete the upgrade. We've provided a sample resource spec here that can be used as an example:
+
+https://github.com/Azure/aks-engine/raw/master/docs/topics/calico-3.3.1-cleanup-after-upgrade.yaml
+
+There are some placeholder tokens in the above `yaml` file, so please reconcile those with the actual values in your cluster. Look for these placeholder strings in the spec, then compare with the running spec of the comparable pre-3.5 resource in your cluster, and modify the cleanup spec accordingly:
+
```

Comment:
Since the manifest is only used for cleanup, those changes are actually not required.